core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -965,4 +962,5 @@ private[spark] object Executor {
   // task is fully deserialized. When possible, the TaskContext.getLocalProperty call should be
   // used instead.
   val taskDeserializationProps: ThreadLocal[Properties] = new ThreadLocal[Properties]
+  val MDC = "mdc."
```

Comment:
this is too short... I'm ok to just hardcode it in the code.

Reply:
fine.